Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -107,26 +107,35 @@ extension BrowserViewController: WKNavigationDelegate {
         }
         
         if isSafeBrowsingEnabled() {
-            var isURLSafe = true //By default, we mark the URL as safe, unless the lookup returns otherwise..
+            var safeBrowsingResult: SafeBrowsingResult = .safe
             let semaphore = DispatchSemaphore(value: 0)
-            SafeBrowsingClient.shared.find(url.hashPrefixes()) { isSafe, error in
+            SafeBrowsingClient.shared.find(url.hashPrefixes()) { result, error in
                 defer { semaphore.signal() }
                 
                 if let error = error {
                     log.error(error)
+                    safeBrowsingResult = result
                     return
                 }
                 
-                isURLSafe = isSafe
+                safeBrowsingResult = result
             }
             
             _ = semaphore.wait(timeout: .now() + .seconds(30))
             
-            if !isURLSafe {
+            // Three types of results.. "safe", "dangerous", "unknown"
+            // In order not to accidentally block websites due to an error, if it's status is unknown, we should possibly show a different web-page that lets them know we couldn't determine if the page was safe or not..
```

Comment:
Having three different result types is fine, but we should update this comment to clarify that we are required to treat "unknown" and "safe" as the same thing. [Only "dangerous" can lead to a user-visible warning](https://developers.google.com/safe-browsing/v4/usage-limits#age-of-data-and-usage).

Reply:
Fixed.